db/controllers.js

```diff
@@ -1,7 +1,19 @@
-const { selectTopics } = require("./models");
+const { selectTopics, readDataFile } = require("./models");
 
-exports.getAllTopics = (req, res, next) => {
+exports.getAllTopics = (req, res) => {
   selectTopics().then((topics) => {
     res.status(200).send({ topics });
   });
 };
+
+exports.getDocumentation = (req, res, next) => {
+  readDataFile()
+    .then((data) => {
+      res.status(200).json(data);
+    })
+    .catch((err) => {
+      if (err.message === "File not found") {
+        res.status(404).send({ msg: err.message });
+      }
```

Comment:
for errors we should really avoid invoke res here, and use next() to pass the error to our middleware and create some error handling functions.